### PR TITLE
Added option to skip XI4J window

### PR DESCRIPTION
**Pull request by N1GHT-MAR3**
Added an option in the .ini to skip the "XInput support added!" window that pops up when the game launches.

First time doing this sort of thing, so I can't promise that I didn't fuck anything up. Building from the updated master and copy-pasting from the updated release seem to work though.

### CODE REVIEW

File: JuicedFixes/JuicedFixes.vcxproj [hunk 1]
```diff
@@ -1,174 +1,174 @@
...
-      <DeploymentContent>true</DeploymentContent>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <ClInclude Include="input.h" />
-  </ItemGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-  <ImportGroup Label="ExtensionTargets">
-  </ImportGroup>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (GrimMaple) [inline JuicedFixes/JuicedFixes.vcxproj]: Please revert changes in this file

File: fixes.ini [hunk 2]
```diff
@@ -1,7 +1,8 @@
 [Fixes]
 PatchInput = true
 PatchVirtualMemory = true
-PatchCalendarCrash = ture
+PatchCalendarCrash = true
+ShowXI4JWindow = true
```
Comment (GrimMaple) [inline fixes.ini:5]: This window's initial purpose was to debug that the parch is loaded :) I think the default value sgould be `false`

**Comment by GrimMaple** [comment]
I'm not sure why this has happened, but `dllmain.cpp` is marked as "Binary" now. Please check what might have gone wrong

**Comment by GrimMaple** [comment]
Also, please squash the commits and write an appropriate commit message.

**Review by GrimMaple — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Please adjust some of the things in this PR and I will be glad to merge this into master. If you need any help, feel free to contact me.

**Comment by N1GHT-MAR3** [comment]
Scrapping the fork for now. Might come back to it once I know what I'm doing.

**Pull request closed without merging**